src/components/activitylog.js

```diff
@@ -4,14 +4,21 @@ define(["events", "globalize", "dom", "datetime", "userSettings", "serverNotific
     function getEntryHtml(entry, apiClient) {
         var html = "";
         html += '<div class="listItem listItem-border">';
-        var color = "Error" == entry.Severity || "Fatal" == entry.Severity || "Warn" == entry.Severity ? "#cc0000" : "#00a4dc";
+        if ("Error" == entry.Severity || "Fatal" == entry.Severity || "Warn" == entry.Severity) {
+            var color = "#cc0000";
+            var icon = "notification_important";
+        }
+        else {
```

Comment:
I would combine this with the line above.

Reply:
Thank you for the feedback
Those comments are the best way to improve my JS :)